.gitignore

```diff
@@ -5,8 +5,8 @@ env_user.py
 intro-threatgrid/*.json
 
 # Mission Data
-# mission-data/
-#!mission-data/README.md
+mission-data/
+mission-data/README.md
 
```

Comment:
@JPedro2 Oh, I see the `#!` for the README.md file means "except don't ignore this file?" It's weird, Git itself chose which way to go on that one. I'll add the `#!` back in.

Reply:
I mean I'll add `!` so that the mission-data/README.md is allowed through.

Reply:
looks good to me, we will ignore the README.md too but that doesnt matter to be honest